docs/ref-manual/approx/array-hashes.md

```diff
@@ -0,0 +1,44 @@
+Array hashes
+============
+
+### Background: Non-Collision of Hashes is Essential for Storage Integrity
```

Comment:
This should be a level-2 header instead of level-3 (i.e. `##` instead of `###`).  Sphinx complains if you jump levels.